src/backend/src/services/change-requests.services.ts

```diff
@@ -324,6 +326,18 @@ export default class ChangeRequestsService {
       }
     }
 
+    // send a reply to a CR's notifications of its updated status
+    try {
+      const relevantThreads = await prisma.message_Info.findMany({ where: { changeRequestId: foundCR.crId } });
+      relevantThreads?.forEach(async (thread) => {
+        await sendSlackCRStatusToThread(thread.channelId, foundCR.crId, thread.timestamp, accepted);
```

Comment:
move the try, catch into the helper method to send the message to thread that you made. If we're gonna do this, everywhere this is called to clean up the error message, then we should just abstract it away into the helper method